cypress/e2e/users/effective-call-number-column-sortable-closed-loans.cy.js

```diff
@@ -0,0 +1,168 @@
+      (patronGroupResponse) => {
+        testData.patronGroup.id = patronGroupResponse;
+        testData.servicePoint = ServicePoints.getDefaultServicePointWithPickUpLocation();
+        ServicePoints.createViaApi(testData.servicePoint).then(() => {
+          testData.defaultLocation = Location.getDefaultLocation(testData.servicePoint.id);
+          Location.createViaApi(testData.defaultLocation).then((location) => {
+            InventoryInstances.createFolioInstancesViaApi({
+              folioInstances: testData.folioInstances,
+              location,
+            });
+          });
+        });
+        cy.createTempUser([Permissions.uiUsersViewLoans.gui, Permissions.inventoryAll.gui]).then(
+          (userProperties) => {
+            testData.user = userProperties;
+            cy.createTempUser([], testData.patronGroup.name).then((testUserProperties) => {
+              testData.testUser.username = testUserProperties.username;
+              testData.testUser.userId = testUserProperties.userId;
+              testData.testUser.barcode = testUserProperties.barcode;
+              UserEdit.addServicePointViaApi(testData.servicePoint.id, testUserProperties.userId);
+              Checkout.checkoutItemViaApi({
+                itemBarcode: testData.folioInstances[0].barcodes[0],
+                servicePointId: testData.servicePoint.id,
+                userBarcode: testUserProperties.barcode,
+              });
+              CheckInActions.checkinItemViaApi({
+                itemBarcode: testData.folioInstances[0].barcodes[0],
+                servicePointId: testData.servicePoint.id,
+              });
+              Checkout.checkoutItemViaApi({
+                itemBarcode: testData.folioInstances[1].barcodes[0],
+                servicePointId: testData.servicePoint.id,
+                userBarcode: testUserProperties.barcode,
+              });
+              CheckInActions.checkinItemViaApi({
+                itemBarcode: testData.folioInstances[1].barcodes[0],
+                servicePointId: testData.servicePoint.id,
+              });
+              cy.login(testData.user.username, testData.user.password, {
+                path: TopMenu.inventoryPath,
+                waiter: InventoryInstances.waitContentLoading,
+              });
+              InventorySearchAndFilter.switchToItem();
+              InventorySearchAndFilter.searchByParameter(
+                'Barcode',
+                testData.folioInstances[0].barcodes[0],
+              );
+              ItemRecordView.waitLoading();
+              ItemRecordView.openItemEditForm(testData.folioInstances[0].instanceTitle);
+              ItemRecordEdit.addEffectiveCallNumber(
+                testData.testUser.effectiveCallNumber1.prefix,
+                testData.testUser.effectiveCallNumber1.number,
+              );
+              ItemRecordEdit.saveAndClose({ itemSaved: true });
+              ItemRecordView.closeItemEditForm();
```

Comment:
Could you please update item via API? Please do everything in preconditions and postconditions via API if possible